hera_sim/rfi.py

```diff
@@ -264,19 +267,13 @@ def rfi_dtv(fqs, lsts, rfi=None, freq_min=.174, freq_max=.214, width=0.008,
         raise ValueError("strength_std must be float or list with len equal to number of bands")
 
     for band, chnc, strngth, str_std in zip(bands, chance, strength, strength_std):
-        try:
-            fq_ind_min = np.argwhere(band <= fqs)[0][0]
-            fq_ind_max = fq_ind_min + int(width / delta_f) + 1
-            this_rfi = rfi[:, fq_ind_min:min(fq_ind_max, fqs.size)]
-
-            rfis = np.random.uniform(size=lsts.size) <= chnc
-            this_rfi[rfis] += np.atleast_2d(np.random.normal(strngth, str_std, size=np.sum(rfis)) * np.exp(
-                2 * np.pi * 1j * np.random.uniform(size=np.sum(rfis))
-            )).T
-        except IndexError:
-            # this is only raised if fqs.max() < bands.min(), which
-            # would mean that DTV wouldn't be observed in any channel
-            pass
+        fq_ind_min = np.argwhere(band <= fqs)[0][0]
+        fq_ind_max = fq_ind_min + int(width / delta_f) + 1
```

Comment:
this line will not be correct if `band < fqs.min()`. In that case, the width should be reduced because only part of the width is overlapping. To get `fq_ind_max` you could do something like:

```
try:
    fq_ind_max = np.argwhere(band + width <=fqs)[0][0]
except IndexError: # the band goes over the upper edge
    fq_ind_max = len(fqs) - 1
```

Reply:
Done.